apps/sms/index.html

```diff
@@ -148,6 +155,33 @@ <h1 data-l10n-id="messages"></h1>
           <a id="threads-settings-button" data-icon="settings" data-l10n-id="settingsButton"></a>
         </gaia-header>
         <article id="threads-container" class="view-body edit-container" data-type="list">
+          <gaia-fast-list id="threads-list" top="0" bottom="0" caching>
+            <!--<template header>
+              <gaia-sub-header data-time="${daySection}">${section}</gaia-sub-header>
+            </template>-->
+            <template item>
```

Comment:
I don't really know how the list works internally, so it's useless to suggest anything :) But do you mean that you want to prevent l10n from mutating the DOM while scrolling or kind of "pause" l10n processing?

One silly idea was to prefix all l10n attributes before inserting element to the list, like ```defer-l10n-id``` and unprefix once list is in idle state starting from elements that are in the viewport - but not sure how ugly or slow it could be :)